modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/servlet/taglib/util/ContentDashboardDropdownItemsProvider.java

```diff
@@ -95,7 +99,30 @@ public ContentDashboardDropdownItemsProvider(
 			() -> {
 				DropdownItem dropdownItem = new DropdownItem();
 
-				dropdownItem.putData("action", "showInfo");
+				dropdownItem.setData(
+					HashMapBuilder.<String, Object>put(
+						"action", "showInfo"
+					).put(
+						"className", contentDashboardItem.getClassName()
+					).put(
+						"classPK", contentDashboardItem.getClassPK()
+					).build());
+
+				ResourceURL resourceURL =
+					_liferayPortletResponse.createResourceURL();
+
+				resourceURL.setResourceID(
+					"/content_dashboard/get_content_dashboard_item_info");
+
+				resourceURL.setParameter(
+					"className", contentDashboardItem.getClassName());
+
+				resourceURL.setParameter(
+					"classPK",
+					String.valueOf(contentDashboardItem.getClassPK()));
```

Comment:
Hey @brunobasto!!

Sorry, I don't understand exactly how should we move it :(

The variable is declared, initialized, and used, maybe there is something in between, but I am not able to see it :(. Could you help me? 

Thanks

Reply:
Replied on the wrong thread: https://github.com/liferay-frontend/liferay-portal/pull/152/files/95849e3afd884d42e223d68361da341e31f20487#r459020921